index.js

```diff
@@ -1,46 +1,32 @@
 "use strict";
 
 const connoisseur = require("./lib/connoisseur");
+const fs = require("fs");
+const sourceFiles = require("yargs").argv._;
 
-const CODE = `
-"use strict";
-
-// lalala
-[4, 9, 16].map(Math.sqrt);  // → [ 2, 3, 4 ]
```

Comment:
Should probably have this sample code around somewhere. May be as a test? (or at least in an `example.js` file)